src/contact.ts

```diff
@@ -477,6 +477,37 @@ export class Contact implements Sayable {
    */
   public alias(empty: null): Promise<boolean>
 
+  /**
+   * GET / SET / DELETE the alias for a contact
+   *
+   * @param {(none | string | null)} newAlias , 
+   * @returns {(string | null | Promise<boolean>)}
+   *
+   * @example GET the alias for a contact
+   * ```ts
+   * const alias = contact.alias()  //@returns {(string | null)}
+   * ```
+   * 
+   * @example SET the alias for a contact
+   * ```ts
+   * const ret = await contact.alias('lijiarui')  //@returns {Promise<boolean>}
```

Comment:
This one is splendid because of your example show code instead of comments.

You can get rid of comments because the code can tell everything.